File: backup/Invest-Invest/Invest-Invest-Repository/src/main/java/com/illunex/invest/invest/entity/JudgeTemplate.java
package com.illunex.invest.invest.entity;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import java.util.Date;

@Entity
public class JudgeTemplate {    // 심사위원
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    Long judgeTemplateIdx;
    Long userIdx;
    Boolean accept;         // 승인여부
    Date inviteDate;        // 초대
    Date createDate;         // 가입일
}

File: api/api/src/main/java/com/illunex/invest/api/core/invest/dto/JudgeTemplate.kt
package com.illunex.invest.api.core.invest.dto

import java.util.*

class JudgeTemplate {
    var judgeTemplateIdx: Long? = null
    var userIdx: Long? = null
    var accept: Boolean? = null             // 승인여부
    var inviteDate: Date? = null            // 초대
    var createDate: Date? = null            // 가입일
}
